lustrefs-exporter/benches/jobstats.rs

```diff
@@ -37,24 +40,42 @@ job_stats:{}"#,
     str_repeat!(JOBSTAT_JOB, 1000)
 );
 
-async fn parse_synthetic_yaml(input: &'static str) {
+async fn parse_synthetic_yaml_otel(input: &'static str) {
+    // Set up OpenTelemetry metrics
+    let registry = Registry::new();
+    let exporter = opentelemetry_prometheus::exporter()
+        .with_registry(registry.clone())
+        .build()
+        .unwrap();
+
+    let provider = SdkMeterProvider::builder().with_reader(exporter).build();
+
+    let meter = provider.meter("test");
+
+    let otel_jobstats = Arc::new(OpenTelemetryMetricsJobstats::new(&meter));
+
     let f = BufReader::with_capacity(128 * 1_024, input.as_bytes());
 
-    let (fut, mut rx) = jobstats_stream(f);
+    let handle =
+        lustrefs_exporter::jobstats::opentelemetry::jobstats_stream(f, otel_jobstats.clone());
 
-    while rx.recv().await.is_some() {}
+    handle.await.unwrap();
```

Comment:
Unwrap()

Reply:
This is benches/tests, unwrap shouldn't be an issue here